docker-compose.yml

```diff
@@ -7,7 +7,6 @@ services:
     network_mode: bridge
     ports:
       - "4566:4566"
-      - "4571:4571"
```

Comment:
Thanks for this PR @kynefuk . Please note that port `4571` is still required - we use it to provision Elasticsearch instances (currently that's the only API that is not routed via the central edge service on port `4566`). Thanks

Reply:
Sorry I didn't notice 4571 is for Elasticsearc.
I made it back to the original ports block.